tests/libnet/console.go

```diff
@@ -0,0 +1,20 @@
+package libnet
+
+import (
+	"fmt"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/tests/console"
+)
+
+func CheckInterface(vmi *v1.VirtualMachineInstance, interfaceName string) error {
```

Comment:
The name is not hinting about what is checked exactly.
Maybe `InterfaceExists`?